poleno/utils/templatetags/poleno/utils.py

```diff
@@ -244,6 +244,7 @@ def with_(value, arg):
         return u'[not callable]'
     return partial(value, arg)
 
+
```

Comment:
Ostal Ti tu zbytocne pridany riadok.

Reply:
PEP8 E302. Ale ok, to nemá zmysel riešiť v týchto zmenách.